client/src/components/title/title.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+
+class title extends Component {
+	constructor(props) {
+		super(props);
+	}
+
```

Comment:
To make `bg-[color]` and `text-[color]` has the same color, I recommend creating an if-statement inside the Title component and then plugging the tailwind value into className. The prop.color will take the color name (orange, blue, red etc..) The color set was added to `tailwind.config`. You can now use `text-orange` to render the orange color

For example
```
if props.color = "orange":
   then line = bg-orange, braces = text-orange
if ...
   ...
```

Reply:
@minhngo3818 I am a little confused here. I understand the idea but not the implementation. How can I connect "line" with the <hr> that is displaying the line? And same for the braces. Would I give an id to the hr called "line"? And then do something like document.getElementById("line").className = "bg-orange"?

Reply:
You can define variables for braces and lines color. Those store tailwind classes for example `text` or `bg`. When the color props is passed. The value concatenates to those variables. Then, pass those variables to className. Another approach is you pass the color name to `className` directly from props. This approach may prompt to error because javascript has no type of safety.